package.json

```diff
@@ -100,41 +100,41 @@
   "license": "MIT",
   "dependencies": {
     "assets-webpack-plugin": "^3.4.0",
-    "babel-cli": "^6.14.0",
-    "babel-core": "^6.14.0",
-    "babel-loader": "^6.2.5",
-    "babel-plugin-transform-runtime": "^6.15.0",
-    "babel-polyfill": "^6.13.0",
+    "babel-cli": "^6.23.0",
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "^6.23.0",
     "babel-preset-es2015": "^6.14.0",
-    "babel-preset-react": "^6.3.13",
+    "babel-preset-react": "^6.23.0",
     "babel-preset-stage-0": "^6.3.13",
-    "babel-register": "^6.14.0",
-    "babel-runtime": "^6.3.19",
+    "babel-register": "^6.23.0",
+    "babel-runtime": "^6.23.0",
     "better-npm-run": "^0.0.14",
-    "css-loader": "^0.26.1",
-    "debug": "^2.2.0",
-    "express": "^4.14.0",
-    "file-loader": "^0.9.0",
+    "css-loader": "^0.26.2",
+    "debug": "^2.6.1",
+    "express": "^4.14.1",
+    "file-loader": "^0.10.1",
     "fs-extra": "^2.0.0",
-    "html-webpack-plugin": "^2.7.1",
-    "imports-loader": "^0.7.0",
+    "html-webpack-plugin": "^2.28.0",
+    "imports-loader": "^0.7.1",
     "ip": "^1.1.2",
     "isomorphic-fetch": "^2.2.1",
     "json-loader": "^0.5.4",
     "phoenixjs": "^1.0.5",
     "react": "^15.3.2",
     "react-dom": "^15.3.2",
     "react-hot-loader": "^3.0.0-beta.6",
-    "react-redux": "^5.0.2",
-    "react-router": "4.0.0-alpha.6",
+    "react-redux": "^5.0.3",
+    "react-router-dom": "^4.0.0-beta.6",
```

Comment:
If you are going to use beta versions please lock to them. Since the iterations are super fast someone getting into this kit at a later time may get accidentally get the wrong version installed and it be unable to come up.